feature/src/main/java/com/teamdontbe/feature/posting/PostingFragment.kt

```diff
@@ -217,6 +188,33 @@ class PostingFragment : BindingFragment<FragmentPostingBinding>(R.layout.fragmen
         }
     }
 
+    private fun FragmentPostingBinding.updateProgress(
+        progressDrawableResId: Int,
+        textLength: Int,
+        backgroundTintResId: Int,
+        textColorResId: Int,
+        clickListener: () -> Unit,
+    ) {
+        pbPostingInput.progressDrawable =
+            context?.drawableOf(progressDrawableResId)
+        pbPostingInput.progress = textLength
+
+        btnPostingUpload.backgroundTintList =
+            ColorStateList.valueOf(
+                ContextCompat.getColor(
+                    root.context,
+                    backgroundTintResId,
+                ),
+            )
+        btnPostingUpload.setTextColor(
+            ContextCompat.getColor(
+                root.context,
+                textColorResId,
+            ),
```

Comment:
```suggestion
            colorOf(textColorResId)
```

`fun Fragment.colorOf(
    @ColorRes resId: Int,
) = ContextCompat.getColor(requireContext(), resId)`

P2 : 요 확장함수 있어서 활용하시면 됩니다!

Reply:
오홍? 아주 조아요옹 적용 완!!